acme/acme/standalone_test.py

```diff
@@ -146,19 +145,22 @@ def tearDown(self):
 
     def test_it(self):
         max_attempts = 5
-        while max_attempts:
-            max_attempts -= 1
+        for attempt in range(max_attempts):
             try:
                 cert = crypto_util.probe_sni(
                     b'localhost', b'0.0.0.0', self.port)
-            except errors.Error:
-                self.assertTrue(max_attempts > 0, "Timeout!")
-                time.sleep(1)  # wait until thread starts
-            else:
                 self.assertEqual(jose.ComparableX509(cert),
                                  test_util.load_comparable_cert(
                                      'rsa2048_cert.pem'))
                 break
+            except errors.Error:
+                self.assertTrue(max_attempts > 0, "Timeout!")
```

Comment:
Now that we never change `max_attempts`, this assertion never fires. Instead, we should change it to something like:
```
self.assertTrue(attempt + 1 < max_attempts, "Timeout!")
```
which will achieve the same effect.